### DISCUSSION TITLE
Is it possible to change default compare tool?

### DISCUSSION THREAD

**Discussion by Wosiu6** [dotnet/ResXResourceManager#658] [Q&A] [answered]
Is it possible to not use the ResXManager compare tool? I just need to get to the source code of the resx file sometimes and I cannot do it most of the time unless I have git changes.

**Comment by tom-englert** [comment, marked as answer]
What do you mean by "compare tool"?
ResXManager is an editor, not a compare tool.

**Reply by Wosiu6** [reply]
Yes, sorry. I say this because often enough whenever I want to see resx files from git changes I get an error that says 'Failed to start the configured compare tool'.
What I need is an option to see the resx files code without using the extension UI. The only functionality I use is auto sorting

**Reply by tom-englert** [reply]
RXRM is not aware of how you have configured your git tools.

**Reply by Wosiu6** [reply]
Okay. Thanks.

Is it possible to open .resx file in Visual Studio to see file's code instead of the extension's editor?

If so, could you tell me how?

**Reply by tom-englert** [reply]
This is the discussion forum of RXRM, if you have questions about basic usage of Visual Studio you should ask in a VS forum
